masterlist.yaml

```diff
@@ -5366,13 +5366,50 @@ plugins:
   - name: 'TTWLODGen.esp'
     url: [ 'https://taleoftwowastelands.com/content/fnvlodgen' ]
     group: *lodGroup
+
   - name: 'tmzLODadditions.esp'
-    url: [ 'https://www.nexusmods.com/newvegas/mods/58562/' ]
+    url:
+    - link: 'https://www.nexusmods.com/newvegas/mods/61206/'
+      name: 'LOD additions and improvements'
     group: *lodGroup
-    after:
-      - 'FNVLODGen.esp'
-      - 'TTWLODGen.esp'
+    msg:
+      - <<: *patch3rdParty
+        subs:
+          - 'Vish''s Patch Hub (FNV-TTW)'
+          - '[ LOD Additions - TTW Patch ](https://www.nexusmods.com/newvegas/mods/77945/)'
+        condition: 'active("tmzLODadditions.esp") and active("TaleOfTwoWastelands.esm") and not active("LOD Additions - TTW Patch.esp")'
+    clean:
+      - crc: 0x0A2432B9
+        util: 'FNVEdit v4.1.4m  EXTREMELY EXPERIMENTAL'
 
+  - name: 'LOD Additions - TTW Patch.esp'
+    url:
+    - link: 'https://www.nexusmods.com/newvegas/mods/77945/'
+      name: 'Vish''s Patch Hub (FNV-TTW)'
+    group: *lodGroup
+    after:  [ 'tmzLODadditions.esp' ]
+    clean:
+      - crc: 0x5BD687F8
+        util: 'FNVEdit v4.1.4m  EXTREMELY EXPERIMENTAL'
+
+  - name: 'NevadaSkies - Ultimate DLC Edition.esp'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/35998' ]
+    group: *weatherGroup
+    inc:
+      - 'NevadaSkies - Basic Edition.esp'
+      - 'NevadaSkies - TTW Edition.esp'
+  - name: 'NevadaSkies - Basic Edition.esp'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/35998' ]
+    group: *weatherGroup
+    inc:
+      - 'NevadaSkies - Ultimate DLC Edition.esp'
+      - 'NevadaSkies - TTW Edition.esp'
+  - name: 'NevadaSkies - TTW Edition.esp'
+    url: [ 'https://www.nexusmods.com/newvegas/mods/35998' ]
+    group: *weatherGroup
+    inc:
+      - 'NevadaSkies - Basic Edition.esp'
+      - 'NevadaSkies - Ultimate DLC Edition.esp'
```

Comment:
This might have been copied over on a rebase or merge.
It doesn't look related to the other changes.

```
  - name: 'NevadaSkies - Ultimate DLC Edition.esp'
    url: [ 'https://www.nexusmods.com/newvegas/mods/35998' ]
    group: *weatherGroup
    inc:
      - 'NevadaSkies - Basic Edition.esp'
      - 'NevadaSkies - TTW Edition.esp'
  - name: 'NevadaSkies - Basic Edition.esp'
    url: [ 'https://www.nexusmods.com/newvegas/mods/35998' ]
    group: *weatherGroup
    inc:
      - 'NevadaSkies - Ultimate DLC Edition.esp'
      - 'NevadaSkies - TTW Edition.esp'
  - name: 'NevadaSkies - TTW Edition.esp'
    url: [ 'https://www.nexusmods.com/newvegas/mods/35998' ]
    group: *weatherGroup
    inc:
      - 'NevadaSkies - Basic Edition.esp'
      - 'NevadaSkies - Ultimate DLC Edition.esp'
```

Reply:
yeah it's a merge, didn't expect that to somehow end up in it, very weird. I'll have to redo it again :(

Reply:
Well you add a commit removing it, and then I can squash the changes when merging.
The rest of the changes seem fine, though I would generally place 3rd party patches into their own section.

Reply:
> Well you add a commit removing it, and then I can squash the changes when merging. The rest of the changes seem fine, though I would generally place 3rd party patches into their own section.

Removing it how?

Reply:
I removed it using the suggestion.

![image](https://github.com/loot/falloutnv/assets/1944639/5d64b5ac-e41f-4fc4-bc12-fe36430806af)

Reply:
And then when I merge, I'll squash and remove the extra commit.